src/app/api/clients/stats/route.ts

```
@@ -0,0 +1,201 @@
+                where,
+                _count: true,
+            }),
+        ]);
+
+        const stats = {
+            total: totalClients,
+            active: activeClients,
+            verified: verifiedClients,
+            newInTimeRange: newClients,
+            byStatus: clientsByStatus.reduce((acc, curr) => ({
+                ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)